### PR TITLE
updated calendar for next week

### PR DESCRIPTION
**Pull request by guyroyse**

### CODE REVIEW

File: docs/get-involved/redis-live/index-redis-live.mdx [hunk 1]
```diff
@@ -53,7 +53,6 @@ Past events:
 
 |        Date          |    Time     | Streamers                                             | Show                                                    |
 | :------------------: | :---------: | :---------------------------------------------------- | :------------------------------------------------------ |
-|   Friday, July 22    | 6:00 PM UTC | [Guy Royse][guy]                                      | Web, Whimsy, and Redis Stack: Random Projects with Guy  |
```
Comment (slorello89) [inline docs/get-involved/redis-live/index-redis-live.mdx:56]: was this deletion intentional?

**Review by slorello89 — CHANGES_REQUESTED** [review]

File: docs/get-involved/redis-live/index-redis-live.mdx (hunk 1 above)
Comment (guyroyse) [reply docs/get-involved/redis-live/index-redis-live.mdx:56]: Yes. My Internet crapped out about 15 minutes into the stream.

Comment (slorello89) [reply docs/get-involved/redis-live/index-redis-live.mdx:56]: ahah! gotcha

**Review by slorello89 — APPROVED** [review]
LGTM 👍

**Pull request merged**